dev/lint-python

```diff
@@ -87,27 +91,46 @@ else
     rm "$PYCODESTYLE_REPORT_PATH"
 fi
 
-# stop the build if there are Python syntax errors or undefined names
-flake8 . --count --select=E901,E999,F821,F822,F823 --max-line-length=100 --show-source --statistics
```

Comment:
BTW, it was never a good idea to use external executable program directly without any check.